plugin/trino-hudi/src/test/resources/hudi-testing-data/hudi_stock_ticks_mor/.hoodie/hoodie.properties

```diff
@@ -0,0 +1,29 @@
+#Updated at 2025-01-06T01:29:42.850Z
```

Comment:
Please add README.md file in `hudi_stock_ticks_mor` and explain the steps to reproduce these files.